### DISCUSSION TITLE
What are the numbers after the hex code of the fill colors?

### DISCUSSION THREAD

**Discussion by ooker777** [Ji11ard/Dataview-to-Mermaid-Flowchart#2] [Q&A] [answered]
```
red fill:#f2b5bd88
orange fill:#efc5b388
yellow fill:#f2d9b588
green fill:#bde0bd88
mint fill:#c9e4d688
aqua fill:#c0dede88
blue fill:#cbcde188
purple fill:#d4c0e388
pink fill:#e6c1d788
grey fill:#d3cfcf88
default fill:#88888888
```

**Comment by Ji11ard** [comment, marked as answer]
They're transparency (alpha).   RRGGBBaa.  The default fills have a 50% transparency (so they darken/lighten based on your note background).  

But it's a purely stylistic choice; so feel free to change these to FF instead of 88, or just remove them completely and leave the six RGB hex characters.

**Reply by ooker777** [reply]
so the default of aa is 50?
